voucher/templates/enroll.html

```diff
@@ -10,18 +10,46 @@ <h1>Boeing employee enrollment page</h1>
             <h2>Confirm Course</h2>
             <hr/>
             <p>Please confirm this is the course you are trying to enroll in:</p>
+            {% if messages %}
+                {% for message in messages %}
+                <div{% if message.tags %} class="form-{{ message.tags }}"{% endif %}>{{ message }}</div>
+                {% endfor %}
+            {% endif %}
             <form method="post" enctype="multipart/form-data" class="voucher-form ">
                 {% csrf_token %}
-                <select name="coupon_version" id="coupon_version" classname="form-control">
-                    {% for choice in eligible_choices %}
-                        <option value="{{ choice.0 }}">{{ choice.1 }}</option>
-                    {% endfor %}
-                </select>
+                <div class="flex">
+                    <select name="coupon_version" id="coupon_version" classname="form-control">
+                        <option value="">Choose Course</option>
+                        {% for choice in eligible_choices %}
+                            <option value="{{ choice.0 }}">{{ choice.1 }}</option>
+                        {% endfor %}
+                    </select>
+                    <div id="voucher-error" class="form-error d-none">Please select a course from the list. If your expected course isn't in the list, please contact <a href="https://xpro.zendesk.com/hc/en-us/requests/new" target="_blank" rel="noopener noreferrer">Customer Support</a></div>
```

Comment:
a couple of nitpicks:

1. I should have put a period at the end of this sentence:

2. My length of this line I suggested is way too long. Does the code linter complain about this? Please add at least one line break. 

```suggestion
                    <div id="voucher-error" class="form-error d-none">Please select a course from the list. 
                        If your expected course isn't in the list, please contact 
                        <a href="https://xpro.zendesk.com/hc/en-us/requests/new" target="_blank" rel="noopener noreferrer">Customer Support</a>.
                    </div>
```

Reply:
Our code linter doesn't work for HTML.